CHANGELOG.md

```diff
@@ -6,10 +6,10 @@ Monitor the release status by regions at [AKS-Release-Tracker](https://releases.
 
 ### Announcements
 
-* AKS will be upgrading the KEDA addon to KEDA 2.15 on AKS clusters with K8s versions >=1.31. KEDA 2.15 brings multiple breaking changes. The breaking changes are listed below:
-  * The removal of [Pod Identity support](https://github.com/kedacore/keda/issues/5035). If you use pod identity, we recommend you move over to [workload identity for your authentication](https://learn.microsoft.com//azure/aks/keda-workload-identity). 
-  * The removal of [Azure Data Explorer 'metadata.clientSecret' as it was not safe for managing secrets](https://github.com/kedacore/keda/issues/4514).
-  * Removal of the [deprecated metricName from trigger metadata section](https://github.com/kedacore/keda/issues/4240). If you are using metricName today, Please use trigger.name to optionally name your trigger. 
+* AKS will be upgrading the KEDA addon to KEDA 2.15 on AKS clusters with K8s versions >=1.31. The AKS team has added KEDA 2.14 to AKS v1.30. KEDA 2.15 and KEDA 2.14 will introduce multiple breaking changes. The breaking changes are listed below:
+  * **KEDA 2.15**: The removal of [Pod Identity support](https://github.com/kedacore/keda/issues/5035](https://github.com/kedacore/keda/issues/5035)). If you use pod identity, we recommend you move over to [workload identity for your authentication](https://learn.microsoft.com//azure/aks/keda-workload-identity). 
+  * **KEDA 2.14**: The removal of [Azure Data Explorer 'metadata.clientSecret' as it was not safe for managing secrets](https://github.com/kedacore/keda/issues/4514).
+  * **KEDA 2.14**: Removal of the [deprecated metricName from trigger metadata section](https://github.com/kedacore/keda/issues/4240).  The two impacted Azure Scalers are Azure Blob Scaler and Azure Log Analytics Scaler. If you are using `metricName` today, please move `metricName` outside of trigger metadata section  to`trigger.name` in the trigger section to optionally name your trigger. To view an example of what this would look like, please view the open [GitHub issue](https://github.com/Azure/AKS/issues/4471).
```

Comment:
```suggestion
  * **KEDA 2.14** for Kubernets = 1.30: The removal of [Azure Data Explorer 'metadata.clientSecret' as it was not safe for managing secrets](https://github.com/kedacore/keda/issues/4514).
  * **KEDA 2.14** for Kubernets = 1.30: Removal of the [deprecated metricName from trigger metadata section](https://github.com/kedacore/keda/issues/4240). The two impacted Azure Scalers are Azure Blob Scaler and Azure Log Analytics Scaler. If you are using `metricName` today, please move `metricName` outside of trigger metadata section  to`trigger.name` in the trigger section to optionally name your trigger. To view an example of what this would look like, please view the open [GitHub issue](https://github.com/Azure/AKS/issues/4471).
```